dfinity/nns-dapp
# Actionable universe navigation

# Motivation

The `Actionable Proposals` page should be shown when the user clicks `Voting` in the main navigation or when the `Actionable Proposals` card in the universe navigation is clicked. With this PR, we add the card to the universe navigation and update the main navigation link to navigate to “Actionable Proposals” when the user is logged-in.
There should be no actionable proposals visible for logged-out users.

# Changes

- Update main menu proposals link to navigate to "Actionable proposals".
- Add empty component for "Actionable proposals" page.
- Add "Actionable proposals" card to `SelectUniverseList` (Desktop. The card is always visible with the separator).
- Add "Actionable proposals" card to `SelectUniverseDropdown` (Mobile. The card is shown only when selected, w/o the separator).

# Tests

- Add `testId` param to Separator component.
- Add `ENABLE_ACTIONABLE_TAB` to `vites.setup` to rewrite it in the unit tests w/o a error.
- Update mock page store to support boolean parameters values (for `actionable`).
- Add test id to the `Separator` component.
- Add PO for `ActionableProposals` empty page.
- Tested manually that the actionable page is reachable.
- Should display "Actionable proposals" card in 

# Todos

Not yet

# Screenshot

| Mobile | Desktop |
|--------|--------|
| <img width="269" alt="image" src="https://github.com/dfinity/nns-dapp/assets/98811342/fd98fbcb-3d53-4da5-98c5-bebfb32c492d"> | <img width="616" alt="image" src="https://github.com/dfinity/nns-dapp/assets/98811342/3f3b9444-66db-42b7-bebd-c3e163ac3e89"> |
